### PR TITLE
feat(core): add DELETE /configs/jwt-customizer API

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Add `DELETE /configs/jwt-customizer/:tokenType` API

## Testing

Covered by unit tests and integration tests.

## Checklist

- [ ] `.changeset`
- [x] unit tests
- [x] integration tests
- [ ] necessary TSDoc comments

### CODE REVIEW

File: packages/core/src/queries/logto-config.ts [hunk 1]
```diff
@@ -75,6 +77,19 @@ export const createLogtoConfigQueries = (pool: CommonQueryMethods) => {
       `
     );
 
+  const deleteJwtCustomizer = async <T extends LogtoJwtTokenKey>(key: T) => {
```
Comment (simeng-li) [inline packages/core/src/queries/logto-config.ts:80]: nit: feels like could make this query method more general. deleteRowsByKey e.g.

**Review by simeng-li — APPROVED** [review]
overall LGTM

**Review by wangsijie — COMMENTED** [review]
Can we set the token to nullable, then there is no need for "post" and "delete", just "put", if the user want to delete, he can pass `{ "token": null }` to "put" payload

**Comment by darcyYe** [comment]
> Can we set the token to nullable, then there is no need for "post" and "delete", just "put", if the user want to delete, he can pass `{ "token": null }` to "put" payload

The whole JWT-customization-related data is stored as a record in our `logto_configs` table. Besides `tokenSample` we also have `script` (which refers to code snippets), `envVars` and `contextSample` (which is a user-customized context data sample for testing purpose). If the user deletes all these fields, we should delete the record if it is no longer used.

**Pull request merged**